src/kmer_unordered_set.cpp

```diff
@@ -28,26 +30,52 @@ KmerSets::KmerSets(path hap1_kmer_fa_path, path hap2_kmer_fa_path){
     }
 
     // fill the kmer sets 
-    load_file_into_unordered_set(hap1_kmer_fa_path, hap1_kmer_set);
-    load_file_into_unordered_set(hap2_kmer_fa_path, hap2_kmer_set);
+    load_file_into_unordered_set(hap1_kmer_fa_path, hap1_kmer_set, "hap1");
+    load_file_into_unordered_set(hap2_kmer_fa_path, hap2_kmer_set, "hap2");
+}
+
+void KmerSets::get_size_of_kmer_file(path file_path, string hap){
+	// # of kmers by (total file size/size of 2 lines)
+
+	// size of 2 fa lines = size of 1 kmer
+	ifstream fileline(file_path, ios::binary );
+	streampos fsize = 0;
+	string fline;
+	// move 1 kmer = 2 lines through file and store size
+	getline(fileline, fline);
+	getline(fileline, fline);
+    fsize = fileline.tellg() - fsize;
+    // move to end
+    fileline.seekg(0,ios_base::end);
+	// set number of kmer variables
+	if (hap=="hap1"){
+		num_hap1_kmers=fileline.tellg()/fsize;
+		cout << hap << " kmer file path: " << file_path << "\n # kmers: " << num_hap1_kmers << endl;
+	}
+	if (hap=="hap2"){
+		num_hap2_kmers=fileline.tellg()/fsize;
+		cout << hap << " kmer file path: " << file_path << "\n # kmers: " << num_hap2_kmers << endl;
+	}
```

Comment:
This function probably shouldn't take a whole extra parameter 'string hap' just for printing. That print statement could go further down the call stack, wherever that information exists already